file-formats/enho/type/zif_aff_enho_v1.intf.abap

```diff
@@ -13,6 +13,26 @@ INTERFACE zif_aff_enho_v1
       value2 TYPE string,
     END OF ty_filter_values.
 
+  "! <p class="shorttext">Customizing</p>
+  "! Customizing
+  "! $values {@link zif_aff_enho_v1.data:co_customizing}
+  TYPES ty_customizing TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Customizing</p>
+    "! Customizing
+    BEGIN OF co_customizing,
+      "! <p class="shorttext">Customizing supported</p>
+      "! Customizing is supported
+      customizing_supported              TYPE ty_customizing VALUE ' ',
+      "! <p class="shorttext">Customizing not supported</p>
+      "! Customizing is not supported
+      customizing_not_supported              TYPE ty_customizing VALUE 'X',
+      "! <p class="shorttext">Customizing supported (no transport)</p>
+      "! Customizing is supported (without transport)
+      customizing_supported_no_trans              TYPE ty_customizing VALUE 'D',
+      END OF co_customizing.
```

Comment:
Shall we just remove the prefix `customizing_`. If we remove it we get also a better text for the scenarios without transport
```suggestion
      supported              TYPE ty_customizing VALUE ' ',
      "! <p class="shorttext">Customizing not supported</p>
      "! Customizing is not supported
      not_supported              TYPE ty_customizing VALUE 'X',
      "! <p class="shorttext">Customizing supported (no transport)</p>
      "! Customizing is supported (without transport)
      supported_no_transport              TYPE ty_customizing VALUE 'D',
    END OF co_customizing.
```

Reply:
yes, I will adapt it in UIA